### PR TITLE
chore: add uninstall steps to remove a Coder OSS deployment from `docs`

### LINKED ISSUE #2487: Add an uninstall script
### Problem 
The install script is putting files in various folders, which makes it difficult to clean up and remove from a workstation or system

### Definition of Done
,there should be an uninstall script to remove Coder and clean things up.

### PR DESCRIPTION
**Pull request by sharkymark**
This PR outlines steps to uninstall Coder

Fixes #2487

### CODE REVIEW

**Review by ericpaulsen — APPROVED** [review]

File: docs/install/uninstall.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+```console
+sudo rm /usr/local/bin/coder
+```
+
+## Coder as a system service configuration
+
+```console
+sudo rm /etc/coder.d/coder.env
+```
```
Comment (bpmct) [inline docs/install/uninstall.md:29]:
```sh
# Alpine
apk del coder

# Debian / Ubuntu
apt remove coder

# Fedora / CentOS
yum remove coder
```

File: docs/install/uninstall.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Uninstall
+
+This article walks you through how to uninstall your Coder server.
+
+<blockquote class="danger">
+  <p>
+  Uninstalling Coder with the built-in PostgreSQL database will delete the database engine and database. Consider backing up the database directory if you would like to reuse it with a future installation. This does not apply if you have installed your own PostgreSQL database instance.
+  </p>
+</blockquote>
```
Comment (bpmct) [inline docs/install/uninstall.md:9]: @sharkymark is this true? I thought the database will remain even after you remove the binary/package?

**Review by bpmct — APPROVED** [review]
Added some details about packages. Will wait on @ammario before merging

File: docs/install/uninstall.md (hunk 2 above)
Comment (sharkymark) [reply docs/install/uninstall.md:9]: @bpmct where else do you think it stores data? there's a data dir.

When can this PR get approved? @ammario 

image

Comment (bpmct) [reply docs/install/uninstall.md:9]: Ah, I mean if you manually delete the `postgres` folder it removes the DB, but I consider uninstalling deleting the binary or using `apt remove`. I personally find this warning a bit confusing but we can keep or clarify

**Review by ammario — CHANGES_REQUESTED** [review]
I agree the "uninstall" warning is confusing. Most software separates the uninstallation of software with the purging of its data.

**Review by ammario — APPROVED** [review]

**Pull request merged**